go/noise/gaussian_noise_test.go

```diff
@@ -430,10 +430,656 @@ func TestThresholdGaussian(t *testing.T) {
 func TestDeltaForThresholdGaussian(t *testing.T) {
 	for _, tc := range thresholdGaussianTestCases {
 		t.Run(tc.desc, func(t *testing.T) {
-			gotDelta := gauss.(gaussian).DeltaForThreshold(tc.l0Sensitivity, tc.lInfSensitivity, tc.epsilon, tc.noiseDelta, tc.threshold)
-			if math.Abs(gotDelta-tc.thresholdDelta) > 1e-10 {
-				t.Errorf("Got delta: %0.12f, want delta: %0.12f", gotDelta, tc.thresholdDelta)
+			gotDelta := gauss.(gaussian).DeltaForThreshold(tc.l0Sensitivity, tc.lInfSensitivity, tc.epsilon, tc.deltaNoise, tc.threshold)
+			if math.Abs(gotDelta-tc.deltaThreshold) > 1e-10 {
+				t.Errorf("Got delta: %0.12f, want delta: %0.12f", gotDelta, tc.deltaThreshold)
 			}
 		})
 	}
 }
+
+func TestInverseCDFGaussian(t *testing.T) {
+	for _, tc := range []struct {
+		desc           string
+		sigma, p, want float64 // Where p is equal to alpha/2.
```

Comment:
Could you please make these multiple lines? That's the idiomatic way of doing things for test structs.